docs/quick_start.md

```diff
@@ -164,3 +164,31 @@ To merge:
 ```
 git commit -a -m "merge honeycomb latest"
 ```
+
+### 8. Run automated tests
```

Comment:
Do we have tests?

Reply:
Oh, I did see some running.  I think they're the ones here:
https://github.com/brown-ccv/honeycomb/blob/main/src/__tests__/utils.test.js

So not a lot of tests, but I thought it seemed right to run them.

Reply:
Ha! I don't know why I missed them. Looks great!